packages/autocomplete-plugin-tags/src/createTags.ts

```diff
@@ -0,0 +1,72 @@
+import { BaseItem } from '@algolia/autocomplete-core';
+import { createRef } from '@algolia/autocomplete-shared';
+
+import { CreateTagsPluginParams } from './createTagsPlugin';
+import { BaseTag, Tag } from './types';
+
+export type OnTagsChangeParams<TTag> = {
+  prevTags: Array<Tag<TTag>>;
```

Comment:
should this be called  `previous` (to avoid abbreviations in names)

Reply:
You mean `previousTags`?

Reply:
either `previous` or `previousTags`